site/docs/encoding/legend.md

```diff
@@ -85,25 +107,8 @@ To customize legends, a `legend` object in [an encoding channel's definition](en
 }
 ```
 
-To provide themes for all legends, the legends config (`config: {legend: {...}}`) can contain the following properties:
-
-### General
-
-{% include table.html props="cornerRadius,fillColor,orient,offset,strokeColor,strokeDash,strokeWidth,padding" source="LegendConfig" %}
-
-### Gradient
-
-{% include table.html props="gradientLabelBaseline,gradientLabelLimit,gradientLabelOffset,gradientStrokeColor,gradientStrokeWidth,gradientHeight,gradientWidth" source="LegendConfig" %}
-
-### Labels
-
-{% include table.html props="labelAlign,labelBaseline,labelColor,labelFont,labelFontSize,labelLimit,labelOffset,shortTimeLabels" source="LegendConfig" %}
-
-### Symbols
-
-{% include table.html props="entryPadding,symbolColor,symbolType,symbolSize,symbolStrokeWidth" source="LegendConfig" %}
-
-### Title
+To provide themes for all legends, the legends config (`config: {legend: {...}}`) can all [legend properties](#properties) except `direction`, `format`, `orient`, `tickCount`, `title`, `values`, and `zindex`.
```

Comment:
"can all" -> "support all"

- orient is supported in Vega legend.

- Perhaps worth mentioning that  `direction` has configs based on type (`gradientDirection` and `symbolDirection`).

Reply:
Done